### PR TITLE
[Website] <BasicHero /> tertiary link support

### PR DESCRIPTION
**Pull request by EnMod**
This PR adds support for tertiary links to the `<BasicHero />` component.

### CODE REVIEW

File: website/components/basic-hero/style.css [hunk 1]
```diff
@@ -42,6 +50,22 @@
...
+        margin-top: 16px;
+
+        &:not(.background-brand):not([disabled]).brand-consul {
+          color: var(--gray-4);
+
+          & .icon svg [stroke] {
+            stroke: var(--gray-4);
+          }
+        }
+      }
```
Comment (BrandonRomano) [inline website/components/basic-hero/style.css:68]: So this CSS override here could actually be completely avoided if you took advantage of slices in JSX.

Slices are really great for scenarios like this, as they gracefully degrade when the array isn't log enough, as they just return `[]` (which will work as desired when mapped over).

```jsx
<div className="links">
  {links.slice(0, 2).map((link, stableIdx) => {
    {/* Old link logic goes here */}
  })
</div>

{/* I'm not sure yet if this should be inside of outside of links, but outside coud simplify the above css max width*/}
{links.slice(2, 3).map((link, stableIdx) => {
  {/* New link logic goes here, and you can appropriately theme this link consistently */}
})
```

File: website/components/basic-hero/index.jsx [hunk 2]
```diff
@@ -13,26 +14,45 @@ export default function BasicHero({
...
+            {links[2] && (
+              <div className="third-link">
+                <a href={links[2].url}>
+                  <span className="g-type-buttons-and-standalone-links">
+                    {links[2].text}
+                  </span>
+                  <span className="icon">
+                    <InlineSvg
+                      src={
+                        '<svg role="img" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M20 12L14 18M4 12H20H4ZM20 12L14 6L20 12Z" stroke="#76767D" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/></svg>'
```
Comment (EnMod) [inline website/components/basic-hero/index.jsx:48]: Probably best to just save this as an extra file then import, what do you think @BrandonRomano?

Comment (BrandonRomano) [reply website/components/basic-hero/index.jsx:48]: Yup, let's add that as a file in the `basic-hero` directory and import this.

File: website/pages/home/index.jsx [hunk 3]
```diff
@@ -14,15 +14,20 @@ export default function HomePage() {
...
             text: 'Download',
             url: '/downloads',
             type: 'download',
           },
           {
-            text: 'Get Started',
-            url: 'https://learn.hashicorp.com/consul',
-            type: 'outbound',
+            text: 'Try HasiCorp Consul Service on Azure',
+            url: 'https://www.hashicorp.com/products/consul/service-on-azure',
```
Comment (BrandonRomano) [inline website/pages/home/index.jsx:29]:
```suggestion
            url: 'https://learn.hashicorp.com/consul/hcs-azure/deploy',
```

File: website/pages/home/index.jsx [hunk 4]
```diff
@@ -14,15 +14,20 @@ export default function HomePage() {
...
+            type: 'inbound',
```
Comment (BrandonRomano) [inline website/pages/home/index.jsx:30]:
```suggestion
            type: 'outbound',
```

File: website/components/basic-hero/index.jsx [hunk 5]
```diff
@@ -13,26 +15,41 @@ export default function BasicHero({
...
+                    }}
+                    title={link.text}
+                    url={link.url}
+                  />
+                )
+              })}
+            </div>
+            {links[2] && (
+              <div className="third-link">
+                <a href={links[2].url}>
```
Comment (BrandonRomano) [inline website/components/basic-hero/index.jsx:42]:
```suggestion
                <a href={links[2].url} rel="noopener" target="_blank">
```

Comment (EnMod) [reply website/components/basic-hero/index.jsx:42]: Right! Good call

**Comment by BrandonRomano** [comment]
Nice work @EnMod, thanks for this 🚀

**Comment by BrandonRomano** [comment]
Actually, closing in favor of #8076, I need to squash these commits so getting this onto `stable-website` goes smoothly.

**Pull request closed without merging**